lib/data_update_scripts/20210921235050_insert_forem_connect_broadcast_message.rb

```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class InsertForemConnectBroadcastMessage
+    def run
+      return if Broadcast.find_by title: "Welcome Notification: forem_connect"
```

Comment:
Super nitpick but we generally only use the paren-less form for DSL-like methods.

```suggestion
      return if Broadcast.find_by(title: "Welcome Notification: forem_connect")
```